src/crudeditor-lib/middleware/notifications.js

```diff
@@ -145,7 +140,29 @@ const eventsMiddleware = ({ context, modelDefinition }) => store => next => acti
     default:
   }
 
-  return next(action)
+  if ([
```

Comment:
Unify active.type handling (vs. current **switch* and **if..else..**) for the sake of clarity in *action.type* maintenance (ex. addition)